src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -8698,6 +8698,32 @@ instruct divL_rReg(rax_RegL rax, rdx_RegL rdx, no_rax_rdx_RegL div,
   ins_pipe(ialu_reg_reg_alu0);
 %}
 
+instruct udivI_rReg(rax_RegI rax, no_rax_rdx_RegI div, rFlagsReg cr, rdx_RegI rdx)
```

Comment:
I suggest to follow the pattern in other `div/mod` instructions: `(rax_RegI rax, rdx_RegI rdx, no_rax_rdx_RegI div, rFlagsReg cr)`

Similar in following new instructions.

Reply:
Pls see the new commit which fixed the pattern.